_pages/mobile-interview-project.md

```diff
@@ -193,17 +195,16 @@ Please do not use code from previous/current employers, for IP / legal reasons :
 
  * Q) **What programming languages can I use?**
 
-Please utilize Java or Kotlin for Android, or Objective-C or Swift for iOS. For iOS, if you need to drop down to CoreFoundation, Accelerate, etc, C is fine too (but you shouldn’t need to).
-
+Please utilize Java or Kotlin for Android, or Objective-C or Swift for iOS.
 Do not feel pressured or required to use Swift or Kotlin if you’ve primarily worked in Objective-C or Java in the past – use whatever you’re most comfortable with! For context, Square still has plenty of Objective-C and Java, and will for some time.
 
-Please do not utilize cross-platform SDKs such as React Native, Flutter, Phonegap, Cordova, etc. We do not utilize these at Square.
+For this project, please do not utilize cross-platform SDKs such as React Native, Flutter, Phonegap, Cordova, etc.
```

Comment:
+1

Reply:
But we don't say that Square doesn't use these? Is there a contradiction? We're just saying not to. Maybe there's some tension because we said above that we use Java and Objective-C to make them feel okay with using them, too. I was already tempted to take that confession out, so maybe this is more motivation?

Reply:
I think we can specifically list out the SDKs we don't use and definitely do not want to see, and say we are in an exploratory phase with KMP explicitly and would recommend against using it.

Reply:
I don't think we should strive to enter into a more lengthy discussion about the current and future states of Square's technical directions and culture here - can we keep it simple? To me, directly saying "don't use cross-platform frameworks" accomplishes our goal: we don't want to review cross platform code, because we want to evaluate native development skills.  

Saying "square does this" or "square does that" to me seems like it crosses a threshold to us requiring that we provide a justification to each of our asks for the candidate. We shouldn't be in that business with this document imo. It leads to the question "what else does square do or not do i don't know about" - and that's very ambiguous from this doc.